submittal/src/main/java/com/kyriba/submittal/domain/Registration.java

```diff
@@ -4,17 +4,32 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+
 
 /**
  * @author M-ABL
  */
+@Entity
 @NoArgsConstructor
 @Getter
 @Setter
 public class Registration
 {
+  @Id
+  @Column(name = "speaker_id")

```

Comment:
It is better to use common practice and name ID for table identifier.

Reply:
what's the common practice? just 'id'?

Reply:
Yes, 'something_id' is used as a reference to 'something' table. This is why such naming is confusing.

Reply:
clear for an ordinal id. Here I use MapsId > so 'id' == speaker.id